tests/Unit/Models/QuestionTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+namespace Engelsystem\Test\Unit\Models;
+
+use DMS\PHPUnitExtensions\ArraySubset\ArraySubsetAsserts;
+use Engelsystem\Models\User\User;
+use Engelsystem\Test\Unit\HasDatabase;
+use Engelsystem\Test\Unit\TestCase;
+use Engelsystem\Models\Question\Question;
+
+class QuestionTest  extends TestCase
+{
+    use ArraySubsetAsserts;
+    use HasDatabase;
+
+    protected $data = [
+        'question' => 'lorem',
+        'answer'   => 'ipsum',
+        'user_id'  => '1',
+    ];
+
+    /**
+     * Prepare test
+     */
+    protected function setUp(): void {
+        parent::setUp();
+        $this->initDatabase();
+    }
+
+    /**
+     * @covers \Engelsystem\Models\Question\Question
+     */
+    public function testBelongsToRelations() {
+
+        /** @var User $user */
+        $user = new User([
+                'name' => 'foo',
+                'password' => '',
+                'email'    => 'foo@bar.batz',
+                'api_key'  => '',
+            ]
+        );
+        $user->save();
+
+        /** @var User $user */
+        $answerUser = new User([
+            'name' => 'bar',
+            'password' => '',
+            'email'    => 'bar@bar.batz',
+            'api_key'  => '',
+        ]);
+        $answerUser->save();
+
+        $question = new Question($this->data);
+        $question->user()
+            ->associate($user)
+            ->save();
+
+        $question->answerUser()->associate($answerUser)->save();
+        $this->assertEquals($user->name, $question->user->name);
```

Comment:
I normally put an empty line between the bootstrapping and testing but thats only my opinion on formatting ^^